cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/config/reader/TemplateSetConfigurationManager.java

```diff
@@ -1,35 +1,36 @@
 package com.devonfw.cobigen.impl.config.reader;
 
+import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Stream;
 
 import com.devonfw.cobigen.api.constants.ConfigurationConstants;
 import com.devonfw.cobigen.api.exception.InvalidConfigurationException;
-import com.devonfw.cobigen.impl.config.entity.io.TemplateSetConfiguration;
+import com.devonfw.cobigen.api.util.TemplatesJarUtil;
+import com.devonfw.cobigen.impl.util.FileSystemUtil;
 
 /**
- * TODO khucklen This type ...
+ * Managing configurations, everything comes together here (I get more specific, once the ticket is done)
```

Comment:
Please get more specific as the ticket needs to get done.

Reply:
whoops 

✓